### PR TITLE
test: basic unit tests for GetComposes

### PR DESCRIPTION
**Pull request by jabia99**
Added basic tests to GetComposes function on server.go

### CODE REVIEW

**Review by atodorov — DISMISSED** [review]
LGTM

File: internal/v1/server_test.go [hunk 1]
```diff
@@ -424,6 +424,50 @@ func TestGetComposeMetadata404(t *testing.T) {
...
+		require.NoError(t, err)
+
+	}))
+	defer api_srv.Close()
+
+	// insert a compose in the mock database
+	dbase := tutils.InitDB()
+	err := dbase.InsertCompose(UUIDTest, "500000", "000000", json.RawMessage("{}"))
+	require.NoError(t, err)
+	created_at, _ := dbase.GetCompose(UUIDTest, "500000")
```
Comment (Gundersanne) [inline internal/v1/server_test.go:450]: - camelcase variables
- check the error here as well
- GetCompose returns a ComposeEntry, so the variable name is a bit confusing.

**Review by Gundersanne — CHANGES_REQUESTED** [review]
Thank you! Sorry for leaving this so long, minor comment.

File: internal/v1/server_test.go [hunk 2]
```diff
@@ -424,6 +424,50 @@ func TestGetComposeMetadata404(t *testing.T) {
...
+	}
+
+	api_srv := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		w.Header().Set("Content-Type", "application/json")
+
+		err := json.NewEncoder(w).Encode(composes)
+		require.NoError(t, err)
+
+	}))
+	defer api_srv.Close()
```
Comment (Gundersanne) [inline internal/v1/server_test.go:444]: for `/composes` you don't need the api_srv, just pass an empty string to startServer.

File: internal/v1/server_test.go [hunk 3]
```diff
@@ -424,6 +424,50 @@ func TestGetComposeMetadata404(t *testing.T) {
...
+
+	// insert a compose in the mock database
+	dbase := tutils.InitDB()
+	err := dbase.InsertCompose(UUIDTest, "500000", "000000", json.RawMessage("{}"))
+	require.NoError(t, err)
+	created_at, _ := dbase.GetCompose(UUIDTest, "500000")
+
+	srv := startServerWithCustomDB(t, api_srv.URL, "*", "", dbase)
```
Comment (Gundersanne) [inline internal/v1/server_test.go:452]: `startServerWithCustomDB(t, "", "*", "", dbase)`, see above

File: internal/v1/server_test.go (hunk 1 above)
Comment (jabia99) [reply internal/v1/server_test.go]: Thanks, I've fixed almost everything. I'll open another PR to test  query parameters and add negative scenarios

**Review by Gundersanne — APPROVED** [review]
Nice thank you!
